src/models/base/ScholarshipModel.ts

```diff
@@ -0,0 +1,13 @@
+// import firebase from 'firebase/app';
+import FirestoreModel from './FirestoreModel';
+
+export default class ScholarshipModel<
```

Comment:
**ScholarshipModel** is not a base class for anything. I suggest moving this up one folder.